### DISCUSSION TITLE
Editing an Obsidian note with Meld Build

### DISCUSSION THREAD

**Discussion by MolaveM** [meld-cp/obsidian-build#6] [Q&A] [answered]
Hi. I was surprised and very impressed with the demo scripts in this Obsidian plugin, and I can see I've got a bit of a steep learning curve ahead of me before I could leverage this power. 

So may I just ask a quick question: I'm guessing that any script I create with this would not be able to make changes to an existing Obsidian note. Because sandboxed JavaScript. Is that correct? Would I be able to create a new one? 

Or is the plugin used to just output data? 

Thanks.

**Comment by meld-cp** [comment, marked as answer]
HI @MolaveM ,

There's a couple of ways to write back to the current note;  you could use [markers](https://github.com/meld-cp/obsidian-build/blob/master/docs/api.md#markers-markers) or the [IO API](https://github.com/meld-cp/obsidian-build/blob/master/docs/api.md#inputoutput-io) (careful though, the, `await $.io.output( file, content, open? )` call will overwrite the file)
